### ISSUE TITLE
TypeError installing/upgrading clangd

### ISSUE THREAD

**Issue by rbtcollins** [clangd/vscode-clangd#329]
```Failed to install clangd language server: TypeError: Expected signal to be an instanceof AbortSignal
You may want to install it manually.``` when accepting the prompt to upgrade from clang 13 to 14.

**Logs**
Please attach the clangd log from the "Output" window if you can.
If possible, run with `--log=verbose` - note that the logs will include the contents of open files!

Nothing was logged to the clangd window during the upgrade interaction.


**System information**
Clangd version (from the log, or `clangd --version`):

```
I[03:12:00.188] argv[0]: /home/robertc/.vscode-server/data/User/globalStorage/llvm-vs-code-extensions.vscode-clangd/install/13.0.0/clangd_13.0.0/bin/clangd
I[03:12:00.188] argv[1]: -clang-tidy
I[03:12:00.188] argv[2]: -background-index
I[03:12:00.188] argv[3]: -compile-commands-dir=/home/robertc/src/uol-swdesign-cm2010
I[03:12:00.188] argv[4]: -header-insertion=never
```

clangd extension version: v0.1.16



Operating system: Linux - Ubuntu impish

**Comment by rbtcollins** [comment]
I removed the clangd in question, and the same error occurs trying to install rather than upgrade.

**Comment by HighCommander4** [comment]
Where do you see this `TypeError`? What is the full error message? Is there an associated stack trace?

**Comment by rbtcollins** [comment]
![image](https://user-images.githubusercontent.com/499678/166629480-1fe00f67-3180-4250-9f55-1c1adc643e09.png)

is the dialog that prompts, then

![image](https://user-images.githubusercontent.com/499678/166629511-f844ff1c-ab92-40ab-bce4-30a50f167260.png)

**Comment by rbtcollins** [comment]
Oh, more info in case it helps:
VScode (local) version info
![image](https://user-images.githubusercontent.com/499678/166629614-73c0cd03-d092-4651-a5b1-ce5d6d4c01a0.png)

The clangd extension is running in vscode-remote on a Linux VM I develop against

**Comment by HighCommander4** [comment]
If you open `Help --> Toggle Developer Tools`, do you see this `TypeError: Expected signal to be an instance of AbortSignal` somewhere in the console? If so, could you paste it and its associated stack trace?

**Comment by sam-mccall** [comment]
Based on the new screenshot, it looks like this is #325    and 0.1.17 should fix it.

**Comment by rbtcollins** [comment]
<details><summary>This seems to be the relevant console messages.</summary>

```
[Extension Host] glibc is 2.34 min is 2.18
console.ts:137 [Extension Host] /home/robertc/.vscode-server/data/User/globalStorage/llvm-vs-code-extensions.vscode-clangd/install/13.0.0/clangd_13.0.0/bin/clangd  --version output:  clangd version 13.0.0 (https://github.com/llvm/llvm-project d7b669b3a30345cfcdb2fde2af6f48aa4b94845d)
Features: linux+grpc
Platform: x86_64-unknown-linux-gnu

console.ts:137 [Extension Host] Checking for clangd update: available= 14.0.0  installed= 13.0.0
console.ts:137 [Extension Host] glibc is 2.34 min is 2.18
console.ts:137 [Extension Host] Downloading  https://github.com/clangd/clangd/releases/download/14.0.0/clangd-linux-14.0.0.zip  to  /home/robertc/.vscode-server/data/User/globalStorage/llvm-vs-code-extensions.vscode-clangd/download/clangd-linux-14.0.0.zip
log.ts:313   ERR [Extension Host] Failed to install clangd:  TypeError: Expected signal to be an instanceof AbortSignal
	at new Bi (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:39:332346)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:39:334743
	at new Promise (<anonymous>)
	at Function.fn [as default] (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:39:334708)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:3194
	at Generator.next (<anonymous>)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37654
	at new Promise (<anonymous>)
	at Nr (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37451)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:3138
	at ub.<anonymous> (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:239:7547)
	at Generator.next (<anonymous>)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:1:1327
	at new Promise (<anonymous>)
	at Re (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:1:1147)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:239:7464
	at y._withProgress (/home/robertc/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/out/vs/workbench/api/node/extensionHostProcess.js:83:43945)
	at y.withProgress (/home/robertc/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/out/vs/workbench/api/node/extensionHostProcess.js:83:43699)
	at Object.withProgress (/home/robertc/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/out/vs/workbench/api/node/extensionHostProcess.js:99:40824)
	at ub.progress (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:239:7442)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:3092
	at Generator.next (<anonymous>)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37654
	at new Promise (<anonymous>)
	at Nr (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37451)
	at n (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:3010)
	at Object.<anonymous> (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:2317)
	at Generator.next (<anonymous>)
	at d (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37507)
console.ts:137 [Extension Host] Failed to install clangd:  TypeError: Expected signal to be an instanceof AbortSignal
	at new Bi (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:39:332346)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:39:334743
	at new Promise (<anonymous>)
	at Function.fn [as default] (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:39:334708)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:3194
	at Generator.next (<anonymous>)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37654
	at new Promise (<anonymous>)
	at Nr (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37451)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:3138
	at ub.<anonymous> (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:239:7547)
	at Generator.next (<anonymous>)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:1:1327
	at new Promise (<anonymous>)
	at Re (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:1:1147)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:239:7464
	at y._withProgress (/home/robertc/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/out/vs/workbench/api/node/extensionHostProcess.js:83:43945)
	at y.withProgress (/home/robertc/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/out/vs/workbench/api/node/extensionHostProcess.js:83:43699)
	at Object.withProgress (/home/robertc/.vscode-server/bin/dfd34e8260c270da74b5c2d86d61aee4b6d56977/out/vs/workbench/api/node/extensionHostProcess.js:99:40824)
	at ub.progress (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:239:7442)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:3092
	at Generator.next (<anonymous>)
	at /home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37654
	at new Promise (<anonymous>)
	at Nr (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37451)
	at n (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:3010)
	at Object.<anonymous> (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:236:2317)
	at Generator.next (<anonymous>)
	at d (/home/robertc/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.16/out/bundle.js:235:37507)
D @ console.ts:137
```

</details>
